### PR TITLE
Development

### PR DESCRIPTION
**Pull request by cero1903**
Sprint 3

### CODE REVIEW

**Review by madsnedergaard — APPROVED** [review]
Hey team. It's a bit late (it was supposed to be done two weeks ago), so I/we don't have time to do a proper code review anymore.

I'll approve this PR so that you're not blocked and have something to show on Friday.

File: src/AddItemPage.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+          <br/>
+          <p>Amount of points to borrow item  </p>
+          <DropdownPointsSelection/>
+          <br/>
+          <br/>
+          <ItemDescription/>
+          </div>
+
+        <div className='btnContainer'>
+          <Link to ='adddetails'><button>Next</button></Link>
```
Comment (Akons97) [inline src/AddItemPage.js]: I can see that this is just navigation to another page, not really adding an item?

File: src/AddMoreDetailsPage.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    <p>Condition of the item</p>
+    <DropdownConditionSelection
+    />
+    <br/>
+    <LocationForm
+    />
+
+  </div>
+  <div className='btnContainer'>
+  <Link to="/itemadded"><button>Upload Item</button></Link>
```
Comment (Akons97) [inline src/AddMoreDetailsPage.js]: The same applies here

File: src/ItemAdded.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+import Nav from './Nav';
+
+const ItemAdded= () => {  
+    return (
+      
+    <section className='main'>
+    <div className='mainContainer'>
+      <Nav />
+        <h1>Item added!</h1>
+        <h3>Other people can now see and borrow your item!</h3>
```
Comment (Akons97) [inline src/ItemAdded.js:27]: It would be nice to redirect a user to his new item or updated list of all items

File: src/ItemCard.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        <div className='itemimage'>
+        <img className='itemimage'src={bike} alt="" height='140px' width='200px' />
+        </div>
+        
+        <div className='itemtext'>
+          <h2> As New Racing Bike </h2>
+          <h4> 30 Points </h4>
+          <p>Available for 3 weeks</p>
+          <p className='blue'>2400 NV, Copenhagen  </p>
+        </div>
```
Comment (Akons97) [inline src/ItemCard.js:16]: You shouldn't have hardcoded values at this point. Instead use your database and populate item list with real data

File: src/ItemProfile.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+import bike from './images/bike2.jpg';
+import {Link} from 'react-router-dom';
+
+const ItemProfile = () => {
+  return (
+    <section className='main'>
+    <div className="mainContainer">
+      <Nav/>
+      <div className='toppage'>
+        <h1>As New Racing Bike</h1>
```
Comment (Akons97) [inline src/ItemProfile.js]: A new user shouldn't have any items right after it creates an account

**Review by Akons97 — APPROVED** [review]
As Mads said, a bit late but better late than never :-)
I can see your progress and app looks good in general. I can see some nice styling there. Also, app is mobile oriented so it doesn't look good on desktop but that's fine. 
I would like to point out that you still have a lot of hard coded logic there considering how far with development phase we are. Try populating items and user info using real data from database, not just hard coding user's name and items. 
I'll approve it so you can continue working but you still have some work to be done.
Good luck and looking forward your final solution!

**Pull request merged**